spec/services/render_message_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe RenderMessage do
+  it 'renders passed template with passed subscriber' do
+    sub = Fabricate(:subscriber)
+    template = Fabricate(:message, html: '{{name}}, {{email}}')
+    out = described_class.new(template, sub).html
+    expect(out).to include(sub.email)
```

Comment:
I almost feel like this should just be `expect(out).to eq ("#{name}, #{email}")` but this works too :smile: